### ISSUE TITLE
"repair unwanted ruptured areas" in definition of Surgeon Role

### ISSUE THREAD

**Issue by sivaramarabandi** [mcwdsi/OMRSE#56]
The current definition of Surgeon Role is  "[superclass: physician role; definition: A physician role borne by a human being and that, if realized, is being realized by its bearer using operative manual and instrumental techniques on a patient to investigate and/or treat a pathological condition such as disease or injury, to help improve bodily function or appearance or to repair unwanted ruptured areas.]"

Is there a particular reason why "repair unwanted ruptured areas" is specifically included here? Perhaps a rewording of the definition would cover it:

       "A physician role realized by its bearer using operative techniques - direct or indirect (laparoscopic, robotic), on a patient to investigate and/or treat (1) a pathological condition such as a disease, disorder or injury, or (2) to help improve bodily function or appearance."

"unwanted ruptured areas" is covered by "disorder". 
Note also the change in "operative manual and instrumental techniques".

**Comment by aellenhicks** [comment]
@sivaramarabandi This definition was developed in the context of the CAFE project with domain experts including surgeons, so I'm not the best person to address this issue. @mbrochhausen what are your thoughts on Sivaram's suggestion?

**Comment by mbrochhausen** [comment]
I agree with some of the aspects Sivaram brought up:
I suggest the following edited version of the definition:
A physician role borne by a human being and that, if realized, is being realized by its bearer using operative manual and instrumental techniques on a patient to investigate and/or treat a disease, disorder or injury, to help improve bodily function or appearance or to repair unwanted ruptured areas.

For the other edits suggested by Sivaram I do not see any arguments in the initial message. I'd be interested to learn about those.

**Comment by mbrochhausen** [comment]
Sorry make that:
A physician role borne by a human being and that, if realized, is being realized by its bearer using operative manual and instrumental techniques on a patient to investigate or treat a disease, disorder or injury, to help improve bodily function or appearance or to repair unwanted ruptured areas.

Also, changes this might need edits to the subclasses of surgeon role necessary.

**Comment by sivaramarabandi** [comment]
@aellenhicks was there any discussion around the use of the wording? As a surgeon, I am curious to know why someone wanted such specifics.

**Comment by mbrochhausen** [comment]
@sivaramarabandi: I think I'll answer that (if Amanda is ok with that). A definition draft was created using wikipedia and medical dictionaries. This draft was reviewed and edited by a group of domain experts in trauma (including surgeons). The assumption that someone requested the specificity you mention is incorrect. Rather, the specificity language seems to be residue from the draft version. Please be aware that none of the surgeons were per se ontologists.

**Comment by aellenhicks** [comment]
Thanks, Mathias. SI aram  what are your thoughts?

Cheers,
Amanda

**Comment by sivaramarabandi** [comment]
You should revisit the definition if you can.

**Comment by aellenhicks** [comment]
In light of this discussion, I have reverted all of the class additions, so they are no longer in the current version of OMRSE. I will move this question to the OMRSE email list. @sivaramarabandi Are you on the list?

**Comment by aellenhicks** [comment]
Followed up on omrse mailing list again. Taking input for the next, then will resolve issue.

**Comment by sivaramarabandi** [comment]
@aellenhicks I am not on the OMRSE email list. How do I get on it?

**Comment by aellenhicks** [comment]
Hi Sivaram,

Here is the link for the Google group.

https://groups.google.com/forum/#!forum/omrse-discuss

Amanda

**Comment by hoganwr** [comment]
The last actionable item here is:

1. Define 'surgeon role' as follows:
A physician role borne by a human being and that, if realized, is being realized by its bearer using operative manual and instrumental techniques on a patient to investigate or treat a disease, disorder or injury, to help improve bodily function or appearance or to repair unwanted ruptured areas.

2. Then review its subclasses to see if edits to its subclasses are necessary as a result of above.

**Comment by hoganwr** [comment]
We discussed on today's (Nov 1, 2023) call dropping the "or to repair unwanted ruptured areas" clause.

We reviewed the subclasses and 'pediatric surgeon role' would also need to have its definition updated. We are working on proposals for new definitions of 'surgeon role', 'pediatric surgeon role', and 'neurosurgeon role' (that last one was a bit circular and also ungrammatical).

**Comment by johnbeve** [comment]
Following the OMRSE call discussion today, I suggest the following definition for 'surgeon role': 

> A physician role that, if realized, is realized by its bearer using operative manual and instrumental techniques to investigate, treat, or modify a patient's anatomical or physiological state for health, functional, or aesthetic purposes.

Note: Physician roles are borne by human beings, hence the removal of "borne by a human being" in the proposed definition. 

Another note: The definition of physician role implies veterinarians cannot bear surgeon roles, since they don't work to promote human health; note sure if that is intended.

**Comment by wdduncan** [comment]
You can also add that it is realized by a surgical procedure. 

Many surgical procedure classes out there.  
Here is one we defined in the [OHD](https://ontobee.org/ontology/OHD?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOHD_0000005) many moons ago:  
>  A series of steps followed in a regular, orderly, definite way, performed by a physician, dentist or surgeon by manual operation.

Here is a definition from [OBIB](https://ontobee.org/ontology/OBIB?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBIB_0000716):  
> A planned process that uses operative manual and instrumental techniques on a patient to investigate and/or treat a pathological condition such as disease or injury, or to help improve bodily function or appearance.

There plenty more ...

**Comment by hoganwr** [comment]
We discussed the current proposal:
A physician role that, if realized, is realized by its bearer using operative manual and instrumental techniques to investigate, treat, or modify a patient's anatomical or physiological state for health, functional, or aesthetic purposes.

One issue is that a surgeon's role is also realized when they do consultations, second opinions, and follow-up care after surgical procedures. So the definition needs to be expanded to include those activities as well.

A new proposal:
A physician role that, if realized, is realized by either (1) performing a surgical procedure, (2) consultation on the potential need for a surgical procedure, (3) giving a second opinion on the need for a surgical procedure, (4) providing post-operative and follow-up care after a surgical procedure, (5) explaining the benefits and risks of a surgical procedure to a patient or patient's parent or otherwise legally authorized representative, (6) obtaining informed consent for a surgical procedure, (7) giving instructions, directions, and communications to healthcare team members before, during, and after the performance of a surgical procedure related to the procedure.

Comment on informed consent: we reviewed OBI's "informed consent process" and like it, and it is applicable to #6 above particularly because it encompasses actually doing the consent and not just signing the papers.  Note that the bearer of the surgeon role is obtaining informed consent for a surgical procedure that they intend to carry out, either as lead surgeon or a member of the surgical team. For example, a surgical resident or fellow might do the informed consent on behalf of the attending and all other members of the surgical team.  There might be multiple surgical specialists involved in a surgical procedure, but informed consent is not necessarily performed by them all (especially if they're brought in unexpectedly to handle unanticipated situations that arise).

Comment on surgical procedure: we reviewed OHD, OBIB, and MAXO definitions of 'surgical procedure' and are not decided on which one "to use" (whether that means we include it specifically in the axiom or not, and we just allude to it).

**Comment by hoganwr** [comment]
Also we want to be abundantly clear that none of those activities is necessarily exclusive to surgeons (e.g. physician assistants). So we want to be explicit that for none of those activities, are we implying that every instance of the activity realizes a surgeon role. We did discuss that it might be possible that every instance of "obtaining informed consent for a surgical procedure" is actually performed by a surgeon and thus realizes a surgeon role.  However, that is not relevant to the definition of 'surgeon role'.

**Comment by sivaramarabandi** [comment]
"A physician role that, if realized, is realized by its bearer using
operative manual and instrumental techniques to investigate, treat, or
modify a patient's anatomical or physiological state for health,
functional, or aesthetic purposes."

This I think would be the 'necessary and sufficient conditions' to define
the role. The rest of it - consultation, second opinion, etc. would all be
inherited from the more general physician role.

best
Sivaram
______________________
Sivaram Arabandi, MD, MS

ONTOPRO

W:  http://ontopro.com/
Ph: 832.726.2322
Li : https://www.linkedin.com/in/sivaramarabandi/

Think Semantics. Tame Silos.

**Comment by hoganwr** [comment]
Sivaram, an excellent point with which I agree. We discussed on today's call that as a subclass of physician role, we also avoid the "circumstantial surgeon" like a layperson doing an emergency tracheotomy.

**Comment by sivaramarabandi** [comment]
Agree... and the 'circumstantial' aspect has potential for use in other
situations such as 'circumstantial pilot' (a passenger without pilot
training flying a plane in an emergency), 'circumstantial soccer coach' (a
parent stepping in when the real coach couldn't make it to the practice),
etc.
